Frontend/android-native/app/src/main/java/in/juspay/mobility/ResourceHandler.java

```diff
@@ -0,0 +1,91 @@
+package in.juspay.mobility;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import in.juspay.hyper.constants.LogCategory;
+import in.juspay.hyper.constants.LogSubCategory;
+import in.juspay.hypersdk.core.JuspayServices;
+
+public class ResourceHandler {
+
+    @NonNull
+    private final Context context;
+    @NonNull
+    private final JuspayServices juspayServices;
+
+    private final String LOG_TAG = this.getClass().getSimpleName();
+
+    public ResourceHandler(@NonNull Context context) {
+        this.context = context;
+        this.juspayServices = new JuspayServices(context,null);
+    }
+
+    protected String getFromAssets (String fileName) {
+        return juspayServices.getFileProviderService().readFromFile(context,fileName);
+    }
+
+    @SuppressLint("DiscouragedApi")
+    protected boolean isResourcePresent (String type, String fileName) {
+        return (context.getResources().getIdentifier(fileName, type, context.getPackageName()) != 0);
+    }
+
+
+    /**
+     * Make sure the file exist before calling this function by isResourcePresent
```

Comment:
In this case can we also have a wrapper within `getRawResource` function?

Reply:
As it is nullable function,  wrapper is not required. and it won't throw any exception, it will return null if resource not found. It is good practice to check if resource present or not that's why mentioned in comment.

Reply:
> It is good practice to check if resource present

Not saying that it's bad practice, wanted to know why can the function itself handle it and return meaning full response instead of having if else whenever this function is used.